docs/en/operation/Troubleshooting.md

```diff
@@ -204,19 +207,40 @@ or [other approaches](https://stackoverflow.com/questions/880557/socket-accept-t
 value in `conf/alluxio-site.properties`. Note that, reducing the value of these two properties may
 potentially add latency for master to serve requests.
 
-TODO(feng) add a section about discover file updates in under fs.
+### Q: I added some files in under file system. How can I reveal the files in Alluxio?
+
+By default, Alluxio loads the list of files the first time a directory is visited. Alluxio will keep using the cached file list regardless of the changes in the under file system. To reveal new files from under file system, you can use the command `alluxio fs ls -f /some/path` to manually discover the new content inside a specific folder. Another way to refresh a directory is to use UFS sync.
```

Comment:
can you wrap the lines more frequently, e.g. around 100 chars? Long lines like this are harder to work with in many editors, and also harder to review changes in.

Reply:
Done.